### ISSUE TITLE
Caret inserts "#" in a list

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#723]
Sometimes, when I'm on a new item in a list and paste something from the clipboard, Caret will prepend it with `#`. For example, if "test" is in my clipboard and I'm here ("|" is a caret location):

```
- a
- b
- |
```

and I press Ctrl+V, I get this:

```
- a
- b
- # test
```

It's not always reproducible but I'm posting a screenshot so that you believe me :)

![image](https://user-images.githubusercontent.com/101152/38556693-d1c309b6-3cca-11e8-8e8e-55dad47481d6.png)

(In my clipboard, there is a string `GlusterFS research #1053`.)

**Comment by astoilkov** [comment]
Can you elaborate a little more? Is this the same as https://github.com/careteditor/caret/issues/722?

**Comment by borekb** [comment]
Sorry, I pressed Enter before filling in any details; the OP is updated now.

**Comment by borekb** [comment]
Oh, is it that Caret now tries to parse HTML to Markdown if it finds HTML in the clipboard? That would be a cool feature if it works reliably :)

**Comment by astoilkov** [comment]
Yes. Exactly. We turned that feature on by default but in the next release, we are moving it to a special command which will be invoked by hitting `Ctrl+Shift+V`.

**Comment by borekb** [comment]
Cool! About the shortcut: Ctrl+Shift+V is commonly "paste as plaintext" so I'd probably consider some other one.

**Comment by borekb** [comment]
Such a cool feature when it works reliably, for example, pasting paragraphs with links is really, really useful. Great idea guys!

**Comment by astoilkov** [comment]
Yes. We will do it with a specific shortcut instead of overriding the default behavior. This way people will choose if they want to paste only the text or the formatted content.
